### ISSUE TITLE
Using class attribute in TextInput's render method limits adding more classes

### ISSUE THREAD

**Issue by mysutka** [atk14/Atk14#11]
I would like to be able add more classes to the default 'text form-control'.

Now the input renders with these default classes when i don't specify them in constructor.
```
...
"widget" => new TextInput()
...
```
When any class is specified in constructor  option attr these default do not render
```
...
"widget" => new TextInput(array(
  "attrs" => array("class" => "any-class"),
))
...
```
This renders input with `class="any-class"` but it should preserve the default classes and render the input with `class="any-class text form-control"`

**Comment by yarri** [comment]
I don't think that preserve default classes is a good idea. There must be a way to set completely different classes. So you should use
```
...
"widget" => new TextInput(array(
  "attrs" => array("class" => "any-class text form-control"),
))
...
```

Does it sound good to you?
